install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -269,3 +276,37 @@ subjects:
   - kind: ServiceAccount
     name: machine-api-operator
     namespace: openshift-machine-api
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s-cluster-machine-api-operator
```

Comment:
This is a total nit, but should this just be `prometheus-k8s-cluster-machine-api-operator`? Not sure where the `-cluster-` comes from.

Reply:
`prometheus-k8s-machine-api-operator`?